scripts/configure-installer-tests-cluster.sh

```diff
@@ -133,3 +134,9 @@ fi
 oc new-project myproject
 sleep 4
 oc version
+
+# KUBECONFIG cleanup only if CI is set
+if [ ! -f $CI ]; then
+    rm -rf $KUBECONFIG
```

Comment:
removing the kubeconfig should happen in the file where it was created (in the test scripts)

Reply:
> removing the kubeconfig should happen in the file where it was created (in the test scripts)

This kubeconfig will not get used in the test script reason https://github.com/openshift/odo/issues/3501#issuecomment-655883217